consensus/src/quorum_store/utils.rs

```diff
@@ -103,14 +103,14 @@ impl MempoolProxy {
         &self,
         max_items: u64,
         max_bytes: u64,
-        return_non_full: bool,
-        exclude_txns: Vec<TransactionSummary>,
+        exclude_txns: Vec<TransactionInProgress>,
     ) -> Result<Vec<SignedTransaction>, anyhow::Error> {
         let (callback, callback_rcv) = oneshot::channel();
         let msg = QuorumStoreRequest::GetBatchRequest(
             max_items,
             max_bytes,
-            return_non_full,
+            true,
```

Comment:
If it is always true can we get rid of the parm?
I think direct_mempool sets it to false, but do we still care about direct mempool? 
Or should we wait until we delete the direct mempool code?

Reply:
Yup, direct_mempool sets it to false, so we need the param until we remove direct_mempool.